packages/react-components/src/components/ParticipantListButton.tsx

```diff
@@ -0,0 +1,188 @@
+   */
+  participantListProps?: ParticipantListProps;
+  /**
+   * Styles of ParticipantList container
+   */
+  participantListContainerStyle?: IStyle;
+  /**
+   * Call Invite URL
+   */
+  callInviteURL?: string;
+  /**
+   * CallBack to mute all remote participants
+   *
+   * @param userId - ID of user requesting to mute all remote participants
+   */
+  onMuteAll?: (userId?: string) => void;
+}
+
+/**
+ * Generates default remote participants subMenuprops for a ParticipantListButton if the props contain participantListProps.
+ *
+ * @param props ParticipantListButtonProps
+ * @returns an array of IContextualMenuItem
+ */
+const generateDefaultRemoteParticipantsSubMenuProps = (props: ParticipantListButtonProps): IContextualMenuItem[] => {
+  const { participantListProps, onMuteAll } = props;
+  const items: IContextualMenuItem[] = [];
+
+  if (participantListProps && participantListProps.participants.length > 0) {
+    const remoteParticipantListProps = { ...participantListProps };
+
+    if (participantListProps.myUserId) {
+      remoteParticipantListProps.participants = participantListProps.participants.filter(
+        (p) => p.userId !== participantListProps.myUserId
+      );
+    }
+
+    items.push({
+      key: 'remoteParticipantsKey',
+      text: 'Remote Participant list',
+      onRender: () => {
+        return (
+          <Stack className={mergeStyles(defaultParticipantListContainerStyle, props?.participantListContainerStyle)}>
+            <ParticipantList {...remoteParticipantListProps} />
+          </Stack>
+        );
+      }
+    });
+
+    items.push({ key: 'participantsDivider1', itemType: ContextualMenuItemType.Divider });
+
+    if (onMuteAll) {
+      items.push({
+        key: 'muteAllKey',
+        text: 'Mute all',
+        title: 'Mute all',
+        iconProps: { iconName: 'MicOff2' },
+        onClick: () => onMuteAll(participantListProps.myUserId)
+      });
+    }
+  }
+
+  return items;
+};
+
+/**
+ * Generates default menuprops for a ParticipantListButton
+ *
+ * @param props ParticipantListButtonProps
+ * @returns IContextualMenuProps
+ */
+const generateDefaultMenuProps = (props: ParticipantListButtonProps): IContextualMenuProps => {
+  const { participantListProps, callInviteURL } = props;
+
+  const defaultMenuProps: IContextualMenuProps = {
+    items: []
+  };
+
+  if (participantListProps && participantListProps.participants.length > 0) {
+    const participantIds = participantListProps.participants.map((p) => p.userId);
+    let remoteParticipantCount = participantIds.length;
+
+    if (participantListProps.myUserId && participantIds.indexOf(participantListProps?.myUserId) !== -1) {
+      remoteParticipantCount -= 1;
+      defaultMenuProps.items.push({
+        key: 'selfParticipantKey',
+        name: 'In this call'
+      });
+    }
+
+    defaultMenuProps.items.push({
+      key: 'remoteParticipantCountKey',
+      name: `${remoteParticipantCount} people`,
```

Comment:
nitty nit: What if there is only one person?
What if there is nobody except the local participant.

Also, should we stick to the word "4 participants" etc?

Reply:
people was in the Figma design for this